test.sh

```diff
@@ -18,18 +23,66 @@ function main() {
   runHTTPSTests
 }
 
+# Run DAP Enterprise test suite
+function runDAP() {
+  prepareOutputDir
```

Comment:
should this really be done in both tests? I would think you would want to do this only once, and ensure the individual output files had different names

Reply:
this is extra weird anyway because it looks like the output files don't go in the `output` directory!?

https://github.com/cyberark/conjur-api-java/blob/master/Jenkinsfile#L23

Reply:
so I don't even need this step?